### ISSUE TITLE
v0.16.4 has `undefined method rm_f' for File:Class (NoMethodError)` error

### ISSUE THREAD

**Issue by SalvatoreT** [lolcommits/lolcommits#428]
# 🐛 Bug

`lolcommits 0.16.3` works perfectly fine for me, but when I upgraded to `0.16.4`, I started getting this error after the camera lit up for taking a picture.

```
❯ /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/lib/lolcommits/cli/process_runner.rb:40:in `delete_pid': undefined method `rm_f' for File:Class (NoMethodError)
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/lib/lolcommits/cli/process_runner.rb:26:in `block in fork_me?'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/lib/lolcommits/cli/process_runner.rb:24:in `fork'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/lib/lolcommits/cli/process_runner.rb:24:in `fork_me?'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/bin/lolcommits:153:in `capture_lolcommit'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/bin/lolcommits:72:in `block in <class:App>'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/methadone-1.9.5/lib/methadone/main.rb:395:in `call_main'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/methadone-1.9.5/lib/methadone/main.rb:167:in `go!'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/bin/lolcommits:209:in `<class:App>'
        from /Users/sal/.asdf/installs/ruby/3.0.2/lib/ruby/gems/3.0.0/gems/lolcommits-0.16.4/bin/lolcommits:17:in `<top (required)>'
        from /Users/sal/.asdf/installs/ruby/3.0.2/bin/lolcommits:23:in `load'
        from /Users/sal/.asdf/installs/ruby/3.0.2/bin/lolcommits:23:in `<main>'
```

I'm on macOS 14.1.1 (23B81) using an M1 MacBook Pro.

# 🩹 Workaround

I got around this by uninstalling the new lolcommits and installing the previous version.

```
gem uninstall lolcommits
gem install --version 0.16.3 lolcommits
```

**Comment by matthutchinson** [comment]
FYI, released [v0.16.5](https://rubygems.org/gems/lolcommits/versions/0.16.5) just now
Thanks for your help here @khramtsoff 🙇‍♂️👏

**Comment by SalvatoreT** [comment]
Thank you!
